packages/graphql/src/translate/queryAST/utils/filter-concrete-entities-with-on-filter.ts

```diff
@@ -0,0 +1,61 @@
+import type { ConcreteEntityAdapter } from "../../../schema-model/entity/model-adapters/ConcreteEntityAdapter";
+import type { InterfaceEntityAdapter } from "../../../schema-model/entity/model-adapters/InterfaceEntityAdapter";
+import type { UnionEntityAdapter } from "../../../schema-model/entity/model-adapters/UnionEntityAdapter";
+import { isUnionEntity } from "./is-union-entity";
+
+/**
+ * Returns the concrete entities presents in the where [_on] argument,
+ * if the where argument is not defined then returns all the concrete entities of the composite target.
+ **/
+export function filterConcreteEntities(
```

Comment:
This name does not match the file name.

Could we try a more explicit name?

```
getConcreteEntitiesInOnArgumentOfWhere
```

That's a bit verbose, so maybe there is a better name

And update the file so the name is the same

Reply:
Good catch, I've played around with it and I've losed track of the name of it!